src/components/Book.js

```diff
@@ -1,33 +1,101 @@
-import React from 'react';
+import { motion } from 'framer-motion';
+import PropTypes from 'prop-types';
+import { useState } from 'react';
 import { useDispatch } from 'react-redux';
 import { removeBook } from '../redux/books/books';
+import './book.css';
 
-const Book = (props) => {
-  const { title, name, id } = props;
+const Book = ({ id, title, category }) => {
   const dispatch = useDispatch();
-  const handleRemoveBook = () => {
+  const [percentage, setPercentage] = useState(Math.floor(Math.random() * 90));
+
+  const increasePerc = () => {
+    if (percentage < 100) {
+      setPercentage(percentage + 10 <= 100 ? percentage + 10 : 100);
+    }
+  };
+
+  const handleRemove = () => {
     fetch(
       `https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/hU7hRqGMmMkUTHelAE4I/books/${id}`,
       {
         method: 'DELETE',
       },
     );
+
     dispatch(removeBook(id));
   };
+
+  const dropIn = {
+    hidden: {
+      x: '150vh',
+    },
+    visible: {
+      x: '0',
+      transition: {
+        duration: 0.2,
+        damping: 25,
+        stiffness: 500,
+      },
+    },
+    exit: {
+      x: '-150vh',
+    },
+  };
+
   return (
-    <div className="book-container" key={id}>
-      <p>
-        {title}
-        <span> by </span>
-        {name}
-      </p>
-      <button type="button" onClick={handleRemoveBook}>
-        remove
-      </button>
-    </div>
+    <motion.div
+      variants={dropIn}
+      initial="hidden"
+      animate="visible"
+      exit="exit"
+      className="card"
+    >
+      <div className="details">
+        <span className="category">{category}</span>
+        <h3 className="title">{title.split(' , ')[0]}</h3>
+        <p className="author">{title.split(' , ')[1]}</p>
+        <div className="actions">
+          <button type="button">Comments</button>
+          <button type="button" onClick={handleRemove}>
+            Remove
+          </button>
+          <button type="button">Edit</button>
+        </div>
+      </div>
+      <div className="progress">
+        <svg className="svg" viewBox="0 0 100 100">
+          <circle cx="50" cy="50" r="42" strokeDasharray="264 0" />
+          <circle
+            className="bar"
+            cx="50"
+            cy="50"
+            r="42"
+            strokeDasharray={`${percentage * 2.64} ${
+              (100 - percentage) * 2.64
+            }`}
+          />
+        </svg>
+        <div className="count">
+          <p>{`${percentage}%`}</p>
+          <span>Completed</span>
+        </div>
+      </div>
+      <div className="chapter">
+        <span>CURRENT CHAPTER</span>
+        <p>Chapter 7</p>
+        <button type="button" onClick={increasePerc}>
+          UPDATE PROGRESS
+        </button>
+      </div>
+    </motion.div>
   );
 };
 
+Book.propTypes = {
+  title: PropTypes.string.isRequired,
+  category: PropTypes.string.isRequired,
+  id: PropTypes.string.isRequired,
+};
+
 export default Book;
```

Comment:
- [ ] Please let's make sure that the styling of our app is EXACTLY the same as the one provided on the Zeplin. Fonts, colors, layout orientations should be pretty much the same as in the original design

Your Design | Zeplin Design
:---:|:---:
![](https://user-images.githubusercontent.com/63087491/156851498-822aa8fc-146b-4a27-8f00-51b259d0c3fb.png) | ![](https://user-images.githubusercontent.com/63087491/156851507-2df2715d-2ded-4b87-ad05-8355d1fd4c70.png)

The header of your app should be distinct from the details page. You forgot to include the user icon. The `div` with the className `count `is not exactly at the center. The vertical line is not so close to the div with the className `chapter`. Also, make sure that the logo is in line to the `div` with the className `details`. For more information, check the annotations on the screenshot above if they may help you.